views/randomBeer.hbs

```diff
@@ -0,0 +1,13 @@
+{{#each responseFromApi}}
+    {{>beerpartial this}}
+    <p>Food pairing:
+        <ul>
+            {{#each this.food_pairing}}
+                <li>
+                    {{this}}
+                </li>
+            {{/each}}
+        </ul>
+    </p>
+    <p>Brewer's tips: {{this.brewers_tips}}</p>
+{{/each}}
```

Comment:
This is correct solution.
Another approach would be to skip keyword `this`, since you are already inside the beer object. So below syntax will give same result

```
{{#each responseFromApi}}
    {{>beerpartial}}
    <p>Food pairing:
        <ul>
            {{#each food_pairing}}
                <li>
                    {{this}}
                </li>
            {{/each}}
        </ul>
    </p>
    <p>Brewer's tips: {{brewers_tips}}</p>
{{/each}}
```